<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1154</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MeshEditor网格编辑</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1154</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>新建</string>
     </property>
     <addaction name="action_New_MeshEdit"/>
     <addaction name="action_New_ManualMeshMatch"/>
     <addaction name="action_New_MeshCut"/>
    </widget>
    <addaction name="menu_4"/>
    <addaction name="separator"/>
    <addaction name="action_Close"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>视图</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="action_New_MeshEdit">
   <property name="text">
    <string>MeshEdit窗口</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="text">
    <string>另存为...</string>
   </property>
  </action>
  <action name="action_Close">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_New_MeshMatch">
   <property name="text">
    <string>MeshMatch窗口</string>
   </property>
  </action>
  <action name="action_New_ManualMeshMatch">
   <property name="text">
    <string>MeshMatch窗口</string>
   </property>
  </action>
  <action name="action_New_MeshCut">
   <property name="text">
    <string>MeshCut窗口</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
